src/core/datatypes/trees/TopologyNode.cpp

```diff
@@ -580,6 +580,14 @@ std::string TopologyNode::computeNewick( bool round )
 /* Build newick string */
 std::string TopologyNode::computePlainNewick( void ) const
 {
+    /* NOTE: Representing a topology as with WITH NO ANNOTATIONS
+     * means that we have to represent sampled ancestors as
+     * out-degree-1 nodes.
```

Comment:
The normal printing routine now prints trees without changing them.  If you want to print trees with 0-length edges in tree monitors you need to change them first using `fnCollapseSA(psi)`.

But this routine is NOT the normal printing routine.  It is used to compute a newick string that represents the topology so that TreeSummary.cpp can do things like `topology_counts[newick]++`.

It is challenging to refactor stuff without breaking things.

Reply:
ah ok, then it is fine 
I was just wondering because the normal printing behaviour was controlled by a global setting before, and I liked that behaviour